python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -102,14 +102,17 @@ def __init__(self,
         self.process_runner = process_runner
         self.event_summarizer = event_summarizer or EventSummarizer()
 
-        # Map from node_id to NodeUpdater processes
+        # Map from node_id to NodeUpdater threads
         self.updaters = {}
         self.num_failed_updates = defaultdict(int)
         self.num_successful_updates = defaultdict(int)
         self.num_failures = 0
         self.last_update_time = 0.0
         self.update_interval_s = update_interval_s
 
+        # Disable NodeUpdater threads if true
+        self.disable_node_updaters = disable_node_updaters
+
```

Comment:
Can you add _why_ this would be set to True?

Reply:
Thanks -- Added an explanation.